wesnoth-wof.po

```diff
@@ -2163,7 +2162,7 @@ msgstr "Nemas"
 #: data/campaigns/Winds_of_Fate/scenarios/04_Journey.cfg:93
 #: data/campaigns/Winds_of_Fate/scenarios/11_Crosswind.cfg:459
 msgid "Shussek"
-msgstr "Šussek"
+msgstr "Sušek"
```

Comment:
Co to? Myslím, že jsem to jméno přepsal dobře - foneticky, se zachovaným syčením uprostřed. Nechci hatit tvou kreativitu a vyžadovat změnu zpátky, jen mne zajímá, co tě k tomu vedlo?

Reply:
Ano, z hlediska věrnosti překladu je to původně správně. Ale nějak mi to přestalo jít přes ústa. Zkoušel jsem si říkat Šussek i variantu Šušek, ale obojí mi přišlo nějak tak klopýtavé či šišlavé. Sušek mi přišlo mnohem více přirozené. Navíc ta zdůrazněná sykavka je to, co by při řeči syčeli hadovití či ještěráci, ale ostaní rasy ne (ty by si to zjednošovali na Šusek). A oslovení té nágy tam v celé kampani provádí jen jedna osoba a to právě není žádný „sykavkovitý“, ale drakonid. Že by drakonid říkal Šussku? Eh...
Ale možná jsem se do toho jen moc zbytečně zahloubal.
Je mi to koneckonců jedno, jen mi to znělo nejpřirozeněji (i proto, že sám znám jednoho Suška).